mender-update/deployments/deployments.cpp

```diff
@@ -141,6 +141,12 @@ error::Error DeploymentClient::CheckNewDeployments(
 			}
 		} else if (status == http::StatusNoContent) {
 			api_handler(CheckUpdatesAPIResponse {nullopt});
+		} else {
+			log::Info(
+				"DeploymentClient::CheckNewDeployments - received unhandled http response: "
+				+ to_string(status));
+			api_handler(expected::unexpected(MakeError(
+				DeploymentAbortedError, "received unhandled http response: " + to_string(status))));
```

Comment:
Nit: `HTTP` instead of `http` - I didn't find any other log message using lowercase `http`.